js/src/views/map/map.js

```diff
@@ -253,6 +253,7 @@ var MapView = Backbone.View.extend({
     this.shareWindowView =  new ShareWindowView(shareOptions)
     this.shareWindowView.render();
     this.shareWindowView.delegateEvents();
+    $("#infowindow-base > .m-modal-window .modal-wrapper").addClass("-share_window");
```

Comment:
We use simple comma in JS (this is a convention between us).

This is not where you should add this class. It is better to do it into the share view. And the most elegant way is to send a param to handlebars and let it handle clases. 
I'm  fixing this and I will show you the solution.